File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ListView/Features/GroupingPage.vb
Namespace CompanionKit.Controls.ListView.Features

	Public Class GroupingPage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

            ' Fill up listView
            Me.mobjListView.Items.Add(New ListViewItem(New String() {"1", "User1", "8-800-1234556", "user1@gmail.com"}))
            Me.mobjListView.Items.Add(New ListViewItem(New String() {"2", "User2", "8-800-9513546", "user2@gmail.com"}))
            Me.mobjListView.Items.Add(New ListViewItem(New String() {"3", "User3", "8-800-8524563", "user3@gmail.com"}))
            Me.mobjListView.Items.Add(New ListViewItem(New String() {"4", "User4", "8-800-9874613", "user4@gmail.com"}))

            ' Define  group for each item
            Me.mobjListView.Items.Item(0).Group = Me.mobjListView.Groups.Item(0)
            Me.mobjListView.Items.Item(1).Group = Me.mobjListView.Groups.Item(1)
            Me.mobjListView.Items.Item(2).Group = Me.mobjListView.Groups.Item(1)
            Me.mobjListView.Items.Item(3).Group = Me.mobjListView.Groups.Item(1)


		End Sub
	End Class

End Namespace

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ListView/Features/GroupingPage.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;

#endregion

namespace CompanionKit.Controls.ListView.Features
{
    public partial class GroupingPage : UserControl
    {
        public GroupingPage()
        {
            InitializeComponent();

            // Fill up listView
            this.mobjListView.Items.Add(new ListViewItem(new string[] { "1", "User1", "8-800-1234556", "user1@gmail.com" }));
            this.mobjListView.Items.Add(new ListViewItem(new string[] { "2", "User2", "8-800-9513546", "user2@gmail.com" }));
            this.mobjListView.Items.Add(new ListViewItem(new string[] { "3", "User3", "8-800-8524563", "user3@gmail.com" }));
            this.mobjListView.Items.Add(new ListViewItem(new string[] { "4", "User4", "8-800-9874613", "user4@gmail.com" }));

            // Define  group for each item
            this.mobjListView.Items[0].Group = this.mobjListView.Groups[0];
            this.mobjListView.Items[1].Group = this.mobjListView.Groups[1];
            this.mobjListView.Items[2].Group = this.mobjListView.Groups[1];
            this.mobjListView.Items[3].Group = this.mobjListView.Groups[1];
        }
    }
}